admin/apple-actions/index/class-push.php

```diff
@@ -110,6 +110,10 @@ private function push() {
 			throw new \Apple_Actions\Action_Exception( __( 'Your API settings seem to be empty. Please fill in the API key, API secret and API channel fields in the plugin configuration page.', 'apple-news' ) );
 		}
 
+		if ( apply_filters( 'apple_news_skip_push', false, $this->id ) ) {
```

Comment:
I'm OK with this. Can you just add a comment above this explaining what it does?